code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -414,3 +418,133 @@
 	. = ..()
 	pizza = new /obj/item/reagent_containers/food/snacks/sliceable/pizza/firecrackerpizza(src)
 	box_tag = "extra spicy pie"
+
+//////////////////////////
+//		Pizza bombs		//
+//////////////////////////
+/obj/item/pizzabox/pizza_bomb
+	var/timer = 10 //Adjustable timer
+	var/timer_set = 0
```

Comment:
While You're Here, think you could set these to booleans if they're used that way?

Reply:
Done!